src/defaultTheme.ts

```diff
@@ -32,6 +32,22 @@ const colors = {
   steelBlue: '#E5EEFE',
   steelBlueDark: '#ABB8C5',
   steelBlueDarker: '#818E9B',
+
+  system: {
```

Comment:
I'm not so fussed on this one, just a suggestion, would something like `ui` be better? Like I say though, disregard if you think not.

Reply:
We can also let Silvia know that these colours can have three variants if that makes life easier - generally, more than 3 are not needed.

Reply:
Also, the theme needs to be verified and signed off on by the 3 of us so happy to tackle any of this stuff in a ticket dedicated to that.

Reply:
I dont think UI is the right word as all components are in the UI. It was called system by silvia in design system so i stuck with that

Reply:
Cool, I don't think 'system' is right either, maybe 'status' - just trying to see what is out there in the wild: https://v2.grommet.io/components